cirq-core/cirq/protocols/decompose_protocol.py

```diff
@@ -119,30 +119,6 @@ def _decompose_(self, qubits: Tuple['cirq.Qid', ...]) -> DecomposeResult:
         pass
 
 
-# pylint: disable=function-redefined
-@overload
-def decompose(
-    val: Any,
-    *,
-    intercepting_decomposer: Optional[OpDecomposer] = None,
-    fallback_decomposer: Optional[OpDecomposer] = None,
-    keep: Optional[Callable[['cirq.Operation'], bool]] = None,
-) -> List['cirq.Operation']:
-    pass
-
-
-@overload
-def decompose(
-    val: Any,
-    *,
-    intercepting_decomposer: Optional[OpDecomposer] = None,
-    fallback_decomposer: Optional[OpDecomposer] = None,
-    keep: Optional[Callable[['cirq.Operation'], bool]] = None,
-    on_stuck_raise: Union[None, TError, Callable[['cirq.Operation'], Optional[TError]]],
-) -> List['cirq.Operation']:
-    pass
```

Comment:
Official answer: they're 100% redundant with the method they're overloading - if anything was calling one of these, it could instead call the remaining `decompose` method with no change in behavior.

Actual answer: `mypy` was complaining about these methods, and I realized they weren't necessary 😛